i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-commercial-extensions.md

```diff
@@ -15,6 +15,29 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
 
 ## Centreon MAP
 
+### 22.04.7 ###
+
+Release date: `soon`
+
+#### New features
+
+- [Editor] You can now edit styles on multiple resources at once.
+- [Editor] You can now manage font properties in the editor.
+- [UI] Made metric link modal labels more explicit by replacing "metric 1" and "metric 2" by "metric in" and "metric out".
+- [Viewer] Resources now have a unique status.
+
```

Comment:
il semble manquer:
[Viewer] You can now define a label for a URL shape. 
MON-19565